tests/func-tests/monitoring_test.go

```diff
@@ -68,8 +68,8 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 				if rule.Alert != "" {
 					Expect(rule.Annotations).To(HaveKeyWithValue("summary", Not(BeEmpty())),
 						"%s summary is missing or empty", rule.Alert)
-					Expect(rule.Annotations).To(HaveKeyWithValue("runbook_url", ContainSubstring(rule.Alert)),
-						"%s runbook_url is missing or doesn't contain alert name", rule.Alert)
+					Expect(rule.Annotations).To(HaveKeyWithValue("runbook_url", HaveSuffix(rule.Alert)),
+						"%s runbook_url is missing or runbook is not equal to alert name", rule.Alert)
```

Comment:
@assafad I think it would need 2 tests here. One for if the runbook_url exists and one if its name is equal to the alert name. Having them together makes fixing the issue harder.

Reply:
This test checks them both. Can you explain what would be harder? I think it's pretty straightforward to check if an alert is missing the `runbook_url` field, or it's named incorrectly. We also do it for all the rest of the fields, for example:
```
Expect(rule.Annotations).To(HaveKeyWithValue("summary", Not(BeEmpty())),
"%s summary is missing or empty", rule.Alert)
``` 
It can fail either if `summary` field doesn't exist, or if it's exists and empty.

Reply:
Updated. Now the workflow is:
- check `runbook_url` exists (and not empty)
- check its suffix
- check the URL is accessible
@sradco Please confirm this sounds good and I'll update the other PRs as well.